kubectl-fdb/cmd/suite_test.go

```diff
@@ -19,6 +19,15 @@ var cluster *fdbv1beta2.FoundationDBCluster
 var clusterName = "test"
 var namespace = "test"
 
+// MockVersionChecker will help to mock the calls to GitHub API to get 'latest' version in tests
+type MockVersionChecker struct {
+	VersionChecker
+}
```

Comment:
```suggestion
type MockVersionChecker struct {
	MockedVersion string
}
```

It would be useful to allow a user to define a specific mocked version for different test cases.